### ISSUE TITLE
Terrasteel Aiot bug

### ISSUE THREAD

**Issue by ArticFox0987** [FTBTeam/FTB-Modpack-Issues#5734]
### Modpack

FTB Skies

### Modpack version

1.6.0

### Launcher

FTB App

### Has the pack been modified

No

### Log Files

[2024-09-07-1.log.gz](https://github.com/user-attachments/files/16919921/2024-09-07-1.log.gz)

### Describe the bug

The botania terrasteel + elementium Aiot is just vanishing with my blocks i break

### Steps to reproduce

1. Use a terrasteel + elementium Aiot 
2. Break a block on survaivel

### Expected behaviour

I expected the block to drop to the ground and then to my inventory

### Screenshots

![Captura de ecrã 2024-09-07 221058](https://github.com/user-attachments/assets/e62fd91c-6ed1-4aaf-8c26-61317ca05c93)
![Captura de ecrã 2024-09-07 221031](https://github.com/user-attachments/assets/7a3d0277-3329-4e5d-9b40-e009b5cf1385)
![Captura de ecrã 2024-09-07 221038](https://github.com/user-attachments/assets/7201adfa-3c6f-4df2-9c79-617fa7a8b019)
![Captura de ecrã 2024-09-07 221049](https://github.com/user-attachments/assets/8b0c3ddc-2adf-4c3d-8afa-b1cc38ae2585)

**Comment by ArtpokeBr** [comment]
Yeah this haven't been fixed for a long time now, I'll completely yeet this Item from JEI.
